server/db/services/campaigns.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
Seems unnecessary to pass in a Model for each method? Considered initializing the module with a model and then using it?